packages/alfa-rules/src/common/predicate/is-clipped.ts

```diff
@@ -37,19 +41,27 @@ export function isClipped(
               )
             ),
             // Or its parent is clipped
-            (node: Node) =>
-              node
-                .parent({
-                  flattened: true,
-                  nested: true,
-                })
-                .some(isClipped(device, context))
+            hasRelevantParent(device, isClipped(device, context))
```

Comment:
```suggestion
            hasRelevantParent(device, isClipped)
```
That needs to call the inner function. Or at least, it can call the inner function which avoids piling up repeated arguments on the call stack.